lib/Target/PTX/PTXTranslation.cpp

```diff
@@ -61,6 +62,27 @@ static bool find_and_replace(std::string &str, const std::string &begin,
 }
 
 static std::string llir_to_ptx(llvm::Module *module, int capability, int ptx) {
+  bool hasExternal = false;
+  for (auto &func : *module) {
+    if (func.hasExternalLinkage()) {
+      hasExternal = true;
+      break;
+    }
+  }
+
+  if (hasExternal) {
+    namespace fs = std::filesystem;
+    static const fs::path libdevice = fs::path(__FILE__)
```

Comment:
could we have a comment here describing what the path is like, it is not quite intuitive reading the C++ code.

Reply:
Let me add.